website/docs/docs/on-premises/releases/1.36.0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 1.36.0
+---
+
+## Release date
+
+17 January 2025
+
+## Features
+
+* New AI user permission - [PACT-PACT-1814](https://smartbear.atlassian.net/browse/PACT-PACT-1814)
+
+## Fixes
+
+* Added validation to publish-provider-contract endpoint to prevent upload of zipped contracts and verification results.
+null
```

Comment:
do you know what these `null`s are about?

Also the script seems to be adding an additional `PACT-` prefix.

Please update the notes to read:

* Added validation to publish provider contract endpoint to prevent invalid data
* Support `x-www-form-urlencoded` validation in bi-directional contract testing

> Added validation and reset to the bidirectional_provider_contract endpoint to capture supported encoding errors and rest the invalid data. 

Is this needed here also? I think the above might be enough to cover it? I'm not clear on what it does so please let me know.

> Fixed a regression whereby new users are unable to login - [PACT-PACT-2948]
This is SaaS only, no need here